data/sql/updates/pending_db_world/rev_1754076991742890400.sql

```diff
@@ -0,0 +1,232 @@
+-- fix low level scourge invasion quests
+-- queststarters / questenders
+UPDATE `creature_template` SET `npcflag` = `npcflag` | 2 WHERE (`entry` IN (16484, 16490, 16493, 16495, 29441));
+
+DELETE FROM `creature_queststarter` WHERE (`quest` IN (9261, 9262, 9263, 9264, 12816)) AND (`id` IN (16484, 16490, 16493, 16495, 29441));
+INSERT INTO `creature_queststarter` (`id`, `quest`) VALUES
+(16484, 9261), -- Lieutenant Nevell
+(16490, 9264), -- Lieutenant Lisande
+(16495, 9262), -- Lieutenant Beitha
+(16493, 9263), -- Lieutenant Dagel
+(29441, 12816); -- Lieutenant Julek
+
+DELETE FROM `creature_questender` WHERE (`quest` IN (9261, 9262, 9263, 9264, 12816)) AND (`id` IN (16484, 16490, 16493, 16495, 29441));
+INSERT INTO `creature_questender` (`id`, `quest`) VALUES
+(16484, 9261), -- Lieutenant Nevell
+(16490, 9264), -- Lieutenant Lisande
+(16495, 9262), -- Lieutenant Beitha
+(16493, 9263), -- Lieutenant Dagel
+(29441, 12816); -- Lieutenant Julek
+
+-- areatrigger
+DELETE FROM `areatrigger_involvedrelation` WHERE `id` IN (4092, 4094, 4095, 4096, 4098, 4099, 4100, 4101, 4103, 4104, 4105, 5151, 5152, 5153, 5154, 5158, 5159, 5160, 5161);
+INSERT INTO `areatrigger_involvedrelation` (`id`, `quest`) VALUES
+(4092, 9260),  -- guid 83048
+(4094, 9260),  -- guid 83044
+(4095, 9260),  -- guid 83049
+(4096, 9260),  -- guid 83046
+(4098, 9261),  -- guid 83047
+(4099, 9261),  -- guid 83045
+(4100, 9265),  -- guid 83041
+-- (4102, 9263),  -- guid 83043 -- too far away from circle somehow
+-- apparently blizzlike bug according to wowhead comments, only 1 circle outside orgrimmar giving quest progress correctly
```

Comment:
It's this those type of bugs, that we do or we do not fix? even if they existed at the time.